index.bs

```diff
@@ -109,6 +113,23 @@ and taking the value of the cumulative layout shift score at that time.
 
 This strategy is illustrated in the usage example.
 
+Source attribution {#source-attribution}
+----------------------------------------
+
+<em>This section is non-normative.</em>
```

Comment:
nit: Do we need to mark all subsections non-normative, given the introduction was already marked?